site/docs/v1.3.0/httpproxy.md

```diff
@@ -1263,6 +1263,35 @@ spec:
             subjectName: foo.marketing
 ```
 
+## Validation of External Client Certificate
+
+It is possible to protect the backend service from unauthorized external clients by requiring the client to present a valid TLS certificate.
+Envoy will validate the client certificate by verifying that a chain of trust can be established from the presented client certificate to the configured trusted root CA certificate.
+Also, all the involved certificates must be unexpired.
```

Comment:
I think it scans a bit better if we combine these sentences:
```
Envoy will validate the client certificate by verifying that it is not expired and that a a chain of trust can be established to the configured trusted root CA certificate.
```

WDYT?

Reply:
Sounds better to me too.